sdks/python/apache_beam/examples/complete/game/user_score.py

```diff
@@ -16,8 +16,9 @@
 #
 
 """First in a series of four pipelines that tell a story in a 'gaming' domain.
-Concepts: batch processing; reading input from Google Cloud Storage and writing
-output to BigQuery; using standalone DoFns; use of the sum by key transform.
```

Comment:
I ended up reimplementing ParseGameEvent as a DoFn to include metrics again, but I moved it to a util.py module where all common functions/classes are.